config.json

```diff
@@ -1,8 +1,8 @@
 {
   "webdriverVersions": {
     "selenium": "2.53.1",
-    "chromedriver": "2.27",
-    "geckodriver": "v0.13.0",
+    "chromedriver": "2.32",
```

Comment:
Sure? I think these get initialized but I believe they are always replaced. This section should be removed soon.